features/shop/account/customer_account/changing_password.feature

```diff
@@ -9,14 +9,14 @@ Feature: Changing a customer password
         And there is a customer "Francis Underwood" identified by an email "francis@underwood.com" and a password "sylius"
         And I am logged in as "francis@underwood.com"
 
-    @todo @ui @api
+    @api @ui @mink:chromedriver
```

Comment:
Is it necessary to add js tag in all these scenarios? I know that this form is now a live component, but we don't use it's features in these scenarios if I see correctly, we just fill in the data and save, so maybe we can skip that? 🤔

Reply:
Unfortunately we have to add js tag. Form is now submitted in component and it wont work without JS